server/difido-server/src/main/java/il/co/topq/report/front/rest/ExecutionResource.java

```diff
@@ -65,6 +67,35 @@ public int post(ExecutionDetails executionDetails) {
 
 	}
 
+	@POST
+	@Path("/update")
+	@Consumes(MediaType.APPLICATION_JSON)
+	@Produces(MediaType.TEXT_PLAIN)
+	public String updateExecutionMetadata(ExecutionMetadataUpdateRequest updateRequest) {
+
+		ExecutionMetadata metadata = metadataProvider.getMetadata(updateRequest.getExecutionId());
+		
+		if (!updateRequest.getExecutionDescription().trim().equals("")) {
+			metadata.setDescription(updateRequest.getExecutionDescription());
+		}
+		
+		metadata.setComment(updateRequest.getExecutionComment());
+		
+		publisher.publishEvent(new ExecutionUpdatedEvent(metadata));
+		return "Successfully updated exection #" + updateRequest.getExecutionId();
+	}
+	
+	@GET
+	@Path("/{executionId}/comment")
+	public String getExecutionComment(@PathParam("executionId") int executionId) {
```

Comment:
I wish that this request can also be removed. We should just add a general **GET** command for the execution and let the client retrieve the field it needs.